test/org/traccar/FilterHandlerTest.java

```diff
@@ -6,15 +6,19 @@
 import static org.junit.Assert.assertNull;
 import org.junit.Before;
 import org.junit.Test;
+import org.traccar.database.IdentityManager;
 import org.traccar.model.Position;
+import org.traccar.model.Device;
 
 public class FilterHandlerTest {
 
+    private EventHandlerTest deviceHandler;
     private FilterHandler filtingHandler;
     private FilterHandler passingHandler;
 
     @Before
     public void setUp() {
+        deviceHandler = new EventHandlerTest();
```

Comment:
You should reuse the code, but it's not the right way to do it. If the functionality is common between two modules, move it in some common place. Possibly a base class.